<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="diplom" type="localizationTest.py" name="agent_localization" output="screen" >
		<param name="map_name" value="map_willowgarage_10_14" type="str" />
		<param name="gazebo_model_name" value="lidar" type="str" />
		
		<param name="hist_pos_limit" value="60" type="int" />
		<param name="hist_orient_limit" value="60" type="int" />

        <param name="laser_count" value="1000" type="int" />
    </node>
	
</launch>
